README.md

```diff
@@ -19,6 +20,28 @@ haversine(lyon, paris, miles=True)
 haversine(lyon, paris, nautical_miles=True)
 # 211.7803748731092  # in nautical miles
 ```
+## Inverse Haversine Formula
+Calculates a point from a given vector (distance and direction) and start point.
+Currently only supports cardinal directions (north, east, south, west).
+
+## Example: Finding arbitary point from Paris
+```python
+from haversine import inverse_haversine
+
+paris = (48.8567, 2.3508) # (lat, lon)
+
+# Finding 32 km west of Paris
+inverse_haversine(paris, 32, 'west')
```

Comment:
I don't think if using only direction is a good idea, it seems to be your use case.

Shouldn't we use an angle in rad starting from the north point
? 
You can then easily use those replacement values:

| direction | angle in rad |
| --- | --- |
| north | 0 |
| east | π * 0.5 |
| south | π | 
| west | π * 1.5 |

You can also maybe expose some constant in the package:

```py
from haversine import WEST;

inverse_haversine(paris, 32, WEST)
```

What do you think ?

Reply:
I like the idea of having directions as constants and those constants being defined as angles.